themes/custom/ts_wrin/sass/components/_programs.scss

```diff
@@ -317,3 +317,25 @@
   @extend .top-border-thick;
   @extend .margin-top-sm;
 }
+
+.report-teaser {
```

Comment:
Is there some css missing possibly? The desktop version doesn't float off to the side like this:
https://www.figma.com/proto/xCkRu4o6Bi5BVmZ4qxxzO3/CK-Editor-templates?page-id=159%3A2213&node-id=159-2216&viewport=399%2C618%2C0.32&t=LG9ZCOAV4qsDroS0-1&scaling=contain&content-scaling=fixed&starting-point-node-id=159%3A2216

Looking at the designs, this may be quite hard to do. We do have a "Callout right" that's close to the same thing, but that is still anchored a little bit within the text. Add to that the complication that on a Publication, the text would ideally show up in the right sidebar, while on something like an article, it will just be off to the side by some "gutter" distance.

Can you think of a way around this? If not, I think using the same logic as the Callout (where the box hangs over a bit but doesn't hang off the side entirely) will be ok.

Reply:
Oooh, I misunderstood the design - I thought there was a sidebar WYSIWYG that this was going to be placed in, that would then be shifted to the body in mobile and tablet view. Let me see if I can adjust this.

Reply:
@mariacha Fixed! Luckily the 'callout' template further down on this page had something fairly close: `/insights/climate-action-progress-1-5-degrees-c`